.github/workflows/bot.yml

```diff
@@ -0,0 +1,97 @@
+on:
+  issue_comment:
+    types: [created, edited]
```

Comment:
Do we need edited type? I think using created type only is simple.

Reply:
This is what I was wondering about. Because there is no strong preference for me, I modified it.